pkg/apis/serving/v1beta1/component.go

```diff
@@ -106,6 +107,10 @@ type ComponentExtensionSpec struct {
 	// More info: http://kubernetes.io/docs/user-guide/annotations
 	// +optional
 	Annotations map[string]string `json:"annotations,omitempty"`
+
+	// The deployment strategy to use to replace existing pods with new ones.
```

Comment:
Can we add a note that this field is only applicable for raw deployment mode

Reply:
Done